.wordlist.txt

```diff
@@ -862,4 +862,7 @@ uUtY
 udL
 img
 href
-OpenSSH
\ No newline at end of file
+OpenSSH
+browsers's
```

Comment:
This is actually a typo, should be "browser's", so you should remove it in the wordlist and correct it in `docs/docs/quickstart/hello-world/README.md`

Reply:
🤦🏻‍♂️ hah yes, thanks